libs/ui/packages/react/package.json

```diff
@@ -75,6 +75,7 @@
   },
   "dependencies": {
     "@floating-ui/react-dom": "^0.4.0",
+    "@ledgerhq/crypto-icons": "1.0.3",
```

Comment:
TODO update to `1.1.0` after https://github.com/LedgerHQ/crypto-icons/pull/43 merged